packages/block-library/src/button/deprecated.js

```diff
@@ -51,6 +51,20 @@ const migrateBorderRadius = ( attributes ) => {
 	};
 };
 
+function migrateAlign( attributes ) {
+	if ( ! attributes.align ) {
+		return attributes;
+	}
+	const { align, ...otherAttributes } = attributes;
+	return {
+		...otherAttributes,
+		className: classnames(
+			otherAttributes.className,
+			'align' + attributes.align
```

Comment:
I think we should use literal template for consistency, how do you think?

`` `align${ attributes.align }` ``

Reply:
Updated 👍